api/package.json

```diff
@@ -74,13 +72,13 @@
     "ejs": "^3.1.8",
     "feathers-authentication-management": "^5.0.0-pre.0",
     "feathers-hooks-common": "^7.0.3",
-    "feathers-locokit": "",
     "feathers-swagger": "^3.0.0",
     "knex": "^2.1.0",
     "knex-schema-inspector": "^3.0.1",
     "koa-helmet": "^6.1.0",
     "koa-ratelimit": "^5.0.1",
     "koa-static": "^5.0.0",
+    "marked": "^4.1.0",
```

Comment:
Are we still use it? If yes, maybe we can use the last version of it.

Reply:
Yup, we are using it for mailing purpose, in the authmanagement system.
I check the latest version, `7.0.2` ! We are late.
I open a new issue for this, as it is not related to build.

see https://github.com/locokit/locokit/issues/247